README.md

```diff
@@ -0,0 +1,87 @@
+![OpenWrt logo](/logo.svg)
+
+OpenWrt Project is a Linux operating system targeting embedded devices. Instead
+of trying to create a single, static firmware, OpenWrt provides a fully
+writable filesystem with package management. This frees you from the
+application selection and configuration provided by the vendor and allows you
+to customize the device through the use of packages to suit any application.
+For developers, OpenWrt is the framework to build an application without having
+to build a complete firmware around it; for users this means the ability for
+full customization, to use the device in ways never envisioned. 
+
+Sunshine!
```

Comment:
OpenWrt is the only project I know using `Sunshine` anywhere, so I intended not to lose such cultural achievements. Apart from that it can also be removed.

Reply:
> cultural achievements

Made my day. :-)